riscof-plugin/sail_cSim/sail_isa.yaml

```diff
@@ -0,0 +1,38 @@
+hart_ids: [0]
+hart0:
+  ISA: RV32IMACZicsr_Zifencei
+  physical_addr_sz: 32
+  supported_xlen: [32]
+  misa:
+    reset-val: 0x0
+    rv32:
+      mxl:
+        implemented: false
+      extensions:
+        implemented: false
+      accessible: true
+    rv64:
```

Comment:
Why is there an rv64 property in a file describing an RV32 configuration?

Reply:
This is a skeleton showing the full schema. The actual YMAL used by a vendor wlll only fill in the parts it needs, but this is here to show what there is to be filled in.